index.html

```diff
@@ -199,6 +199,30 @@ <h4 class="connection2">LET'S CONNECT</h4>
             </div>
         </div>
     </section>
+
+    <section class="contact-form">
+            <h2 class="contact-me">Contact me</h2>
+            <p class="contact-text">
+                If you have an application you are interested in developing,
+                a feature that you need built or a project that needs coding. 
+                I’d love to help with it!
+            </p>
+
+        <form action="https://formspree.io/f/meqwrnzl" method="post">
+            <ul class="contact-list">
+                <li>
+                    <input type="text" id="name" placeholder="Full name" maxlength="30" required />
+                </li>
+                <li>
+                    <input type="email" id="email" placeholder="Email Adress" required />
+                </li>
+                <li>
+                    <textarea rows="6" cols="40" id="text" placeholder="Write your message here..." maxlength="500"></textarea>"
+                </li>
+                <button type="submit" id="contact-submit">Get in touch</button>
```

Comment:
- [ ] Your inputs look good at first glance, yet! i'm somehow not able to submit my entry into the form.
- Kindly check the error message received, and fix it appropriately 👍 

## Error:
![image](https://user-images.githubusercontent.com/50990925/230352855-e4ff90f0-6a03-48ef-b0d0-404e5f682e41.png)